### PR TITLE
UI: Added `VoiceChat` and `TeamVoiceChat`

### PR DESCRIPTION
**Pull request by Alf21**
unknown (4)

TODO:
- [x] Discuss how to display / separate team voice chat and global voice chat

### CODE REVIEW

**Comment by mzegar** [comment]
@Alf21 I'm guessing we are going to keep this up meanwhile the bug report exists?

**Comment by Alf21** [comment]
Hmmm... Currently I can't think of a way fixing it, it has to be done sbox-sided, so we don't need to wait for anything... Let me remove the error printing and adjust the bugreport to a commit link.

Could you test whether you can reproduce this issue (maybe it's caused by my setup),

File: code/player/TTTPlayer.VoiceChat.cs [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                            clients.Add(client);
+                        }
+                    }
+                }
+
+                _oldReceiveClients[player] = clients;
+            }
+            else
+            {
+                // player did not talked before
```
Comment (mzegar) [inline code/player/TTTPlayer.VoiceChat.cs:92]: Just minor grammar, `Player did not talk before`, or `Player has not talked before`

**Review by mzegar — APPROVED** [review]
Seems reasonable enough

**Pull request merged**